### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AardWolf**
Added the remind command. And some other things.

Can't do an automatic merge so I introduced something interesting. -- simple resolution based on the changes to when timers are announced. Will use the dev code.

### CODE REVIEW

File: MHTimer.js [hunk 1]
```diff
@@ -139,6 +146,61 @@ function messageParse(message) {
...
+        case 'remind':
+            if (tokens.length === 1) {
+                message.channel.send("Did you want me to remind you for sg, fg, reset, spill, or cove?");
+            } else {
+                var area = timerAliases(tokens[1].toLowerCase());
+                //confirm it is a valid area
+                var found = 0;
+                for (var i = 0; i < timers_list.length; i++) {
+                    if (timers_list[i].getArea() === area) {
+                        i = timers_list.length;
```
Comment (DevJackSmith) [inline MHTimer.js]: there's a command called break for this.

File: MHTimer.js [hunk 2]
```diff
@@ -139,6 +146,61 @@ function messageParse(message) {
             }
             // console.log(typeof retStr);
             break;
+        case 'remind':
+            if (tokens.length === 1) {
```
Comment (DevJackSmith) [inline MHTimer.js]: i would rename tokens to user_message_parts or something more descriptive in the future

File: .gitignore [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
-timer_settings.json.bak
-MHTimer.log
-MHTimer.pid
+/settings.json
+/project.geany
+/timer_settings.json.bak
+/MHTimer.log*
+/MHTimer.pid
 .test_timers.json
 .timers.json
```
Comment (DevJackSmith) [inline .gitignore]: why are we not putting a slash in front of these two too?

Comment (AardWolf) [inline .gitignore]: I think all "." files should be excluded no matter the directory structure. But putting a / makes sense.

**Pull request merged**